frontend/src/pages/Configure/Create.js

```diff
@@ -97,15 +98,43 @@ function Create() {
   const errors = useState(['app', 'wizard', 'errors', 'create']);
   const pending = useState(['app', 'wizard', 'pending']);
   const editing = getState(['app', 'wizard', 'editing']);
+  const [preferences, setPreferences] = React.useState(
+    undefined
+  );
   return (
     <>
       <Header description={`This is the cluster configuration that will be used to ${editing ? 'update' : 'create'} your cluster.`}
       ></Header>
-      <textarea
-        disabled={ pending ? true : false}
-        spellCheck="false"
-        className="configuration-data" value={clusterConfig}
-        onChange={(e) => {setState(configPath, e.target.value)}} />
+      <CodeEditor
+        ace={window.ace}
+        language="yaml"
```

Comment:
Isn't this duplicated?

Reply:
I think this one is for the particular instance that is being instantiated.